modules/javafx.graphics/src/main/java/javafx/css/converter/ShapeConverter.java

```diff
@@ -43,6 +43,12 @@
 
     public static StyleConverter<String, Shape> getInstance() { return INSTANCE; }
 
+    /**
+     * Creates a default {@code ShapeConverter}.
+     */
+    public ShapeConverter() {
+    }
+
```

Comment:
Looks like a singleton class.

Reply:
Agreed. This is another candidate for removal.